x-pack/plugins/security_solution/cypress/e2e/detection_rules/prebuilt_rules_notifications.cy.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createRuleAssetSavedObject } from '../../helpers/rules';
+import {
+  LOAD_PREBUILT_RULES_BTN,
+  LOAD_PREBUILT_RULES_ON_PAGE_HEADER_BTN,
+  UPDATE_PREBUILT_RULES_CALLOUT,
+} from '../../screens/alerts_detection_rules';
+import { deleteFirstRule, waitForRulesTableToBeLoaded } from '../../tasks/alerts_detection_rules';
+import {
+  installAvailableRules,
+  createNewRuleAsset,
+  preventPrebuiltRulesPackageInstallation,
+} from '../../tasks/api_calls/prebuilt_rules';
+import { cleanKibana, resetRulesTableState, deleteAlertsAndRules } from '../../tasks/common';
+import { esArchiverResetKibana } from '../../tasks/es_archiver';
+import { login, visitWithoutDateRange } from '../../tasks/login';
+import { SECURITY_DETECTIONS_RULES_URL } from '../../urls/navigation';
+
+describe('Detection rules, Prebuilt Rules Installation and Update workflow', () => {
+  before(() => {
+    cleanKibana();
+    login();
+  });
+  beforeEach(() => {
+    /* Make sure persisted rules table state is cleared */
+    resetRulesTableState();
+    deleteAlertsAndRules();
+    esArchiverResetKibana();
```

Comment:
Can we check if the use of this method is strictly necessary? If there is anything else we need to delete that is not present in `cleanKibana()` we should consider adding it to it. This method is slower than `cleanKibana` and can lead us to flakiness.

Reply:
We need this in order to clean the state with the package installation for each test; otherwise, the EPR responds with `already_installed`, but we want to make assertions on a brand new installation, and the tests break.

We could move it to `cleanKibana()` but I think I would need more time to investigate how it will affect other tests and I'd prefer to do that in a separate PR.

Reply:
In that case, can you please load the `audibeat` archive in an `after` hook? that would minimize the risk of flakiness since is the archive we load by default when launching cypress :)

Reply:
Done, thanks

Reply:
@MadameSheema Removing esArchiverResetKibana. Your're right, this doesn't make sense here. It is only necessary in `x-pack/plugins/security_solution/cypress/e2e/detection_rules/prebuilt_rules_install_update_workflows.cy.ts`